.gitignore

```diff
@@ -1,3 +1,7 @@
+# temp files
+on_disk_properties.db
+tmp.py
```

Comment:
These seem very specific for what you are doing, I suggest removing and using global git ignore(https://stackoverflow.com/questions/7335420/global-git-ignore)

Reply:
`on_disk_properties.db` is used by tests. I can move it to `/test/`. I agree regarding the `tmp.py`.